### ISSUE TITLE
Custom changeTriggers

### ISSUE THREAD

**Issue by alsar** [theironcook/Backbone.ModelBinder#171]
I want to change the deafault behavior of the changeTriggers. But I can't make it work.
It works with the default settings `this.modelBinder.bind(this.model, this.el, null, {'changeTriggers': {'': 'click', '[contenteditable]': 'blur'}});`
But when I try to bind the model update to a button click, it doesn't work.
I do it like this: `this.modelBinder.bind(this.model, this.el, null, {'changeTriggers': {'#editCoverConfirm': 'click'}});`
Basically it doesn't work with anything else than the default settings. Can somebody help me to understand how to make this work?
